### ISSUE TITLE
Trying to access another device from the host network from the IoT Edge modules

### ISSUE THREAD

**Issue by sbosman84** [WAGO/azure-iot-edge#2]
Based on the documentation we succeed to deploy the azure-iot-edge runtime on the WAGO device connected it our Azure IoTHub and successfully deployed some modules on the device. 

But when we try to connect to external device via a TCP/UDP connection from a IoT Edge module running on the Wago device we're not able to access the external device. The external device is in the same ip segment as the host network of the Wago device. And also if we deploy an container on the host network of the docker instance we are able to access the external device. 

So we assume that we cannot access the external device because all the containers for the IoT Edge modules are created within a separate azure-iot-edge docker network and probably the routing is not configured properly. Do you perhaps have some guidance on how to approach this scenario? And is this even a supported scenario from this device?

**Comment by DirkMeihoefer** [comment]
We did not tried this scenario ourself. My first idea and quick shot would be to create and configure an azure-iot-edge docker network to use the macvlan driver if that would be acceptable. Macvlan networks allow you to assign a MAC address to a container, making it appear as a physical device on your network. This network can be configured before starting IoT Edge, as the iotedged does a "get or create" operation on the network when starting. Pre-configuring a network and updating the network in the config.yaml allows complete control over its settings.
[Networking on IoT Edge](https://github.com/Azure/iotedge/blob/master/doc/networking.md)

**Comment by IkkertSergei** [comment]
Hello bosman84 . The solution described by colleague Dirk is based on the case that the external devices should access the Iot-Edge modules. In this case Macvlan approach makes sense.  
In the case that the Iot-Edge modules should communicate with an external device in the same network, it should work without any problems. I have already tried Azure IoT Edge Modbus modules and it worked once.
https://github.com/Azure/iot-edge-modbus
 We are currently working on the new version of Iot Edge runtime. This will be the last released version 1.0.10 of iot edge where some bugs have been fixed. The version will probably be released next week on the github. We will check the communication between Iot-Edge modules and external devices again.
